pages_/basePage.py

```diff
@@ -0,0 +1,29 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class BasePage():
+    def _init_(self, driver: webdriver.Chrome):
+        self.driver = driver
+
+    def _find_element(self, by, value):
+        try:
+            element = WebDriverWait(self.driver, 10).until(EC.visibility_of_element_located((by, value)))
+            return element
+        except:
+            print("Error: Element not Found")
+            exit(1)
+
+    def _click(self, webElement):
+        webElement.click()
+
+    def _fill_field(self, webElement, text):
+        webElement.clear()
+        webElement.send_keys(text)
+
+    def _get_title(self):
```

Comment:
input 
        logger("INFO", f"The title is found successfully: {self.driver.title}")